src/minimega/main.go

```diff
@@ -41,6 +42,8 @@ var (
 	f_msaTimeout = flag.Uint("msa", 10, "meshage MSA timeout")
 	f_port       = flag.Int("port", 9000, "meshage port to listen on")
 	f_ccPort     = flag.Int("ccport", 9002, "cc port to listen on")
+	f_vmRate     = flag.Duration("vm-rate", time.Second/4, "average rate to launch KVM VMs")
```

Comment:
this should be in the api instead of a flag - if i boot something and it fails, i'd like to change the metrics without having to restart minimega.